### PR TITLE
Task #8284: Create UI for User Setting Screen

### PR DESCRIPTION
**Pull request by tanbtd-1747**
### Issue Link :link:
https://edu-redmine.framgia.vn/issues/8284

### Goals :soccer:
- Create UI for User Setting Screen

### CODE REVIEW

**Comment by tanbtd-1747** [comment]
ready

File: Money*/Money*/Scenes/Dashboard/DashboardViewController.swift [hunk 1]
```diff
@@ -7,7 +7,41 @@
...
+    
+    // MARK: - Private functions
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addAuthorizationListener()
+    }
+    
+    private func addAuthorizationListener() {
+        Auth.auth().addStateDidChangeListener { [weak self] (_, user) in
+            guard let self = self else { return }
```
Comment (longvt96) [inline Money*/Money*/Scenes/Dashboard/DashboardViewController.swift]: remove

File: Money*/Money*/Scenes/Dashboard/DashboardViewController.swift [hunk 2]
```diff
@@ -7,7 +7,41 @@
...
+            
+            guard let user = user else {
+                return
+            }
+            self.user = User(auth: user)
```
Comment (longvt96) [inline Money*/Money*/Scenes/Dashboard/DashboardViewController.swift]:
```suggestion
            self?.user = User(auth: user)
```

File: Money*/Money*/Scenes/Dashboard/DashboardViewController.swift [hunk 3]
```diff
@@ -7,7 +7,41 @@
...
+        }
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        switch segue.identifier {
+        case Identifier.segueFromDashboardToUserSetting:
+            let userSettingViewController = segue.destination as! UserSettingViewController
```
Comment (longvt96) [inline Money*/Money*/Scenes/Dashboard/DashboardViewController.swift]: k đc `as!` anh nhá

**Review by tgdong2296 — APPROVED** [review]

**Review by longvt96 — APPROVED** [review]

**Review by minhduc2511 — APPROVED** [review]

**Review by tienpx3010 — APPROVED** [review]

**Comment by tienpx3010** [comment]
merged

**Pull request merged**